packages/uows-client-generator/src/construct-functions/constructFunctions.ts

```diff
@@ -85,5 +94,12 @@ export const constructorTemplate = (wsdl: string) => {
                   this.wsdlUrl = '${wsdl}';
               else
                   this.wsdlUrl = wsdlUrl;
+
+              soap.createClient(this.wsdlUrl, (error, client) => {
+                if (error) {
+                  logger.logError('An error occurred while creating the UOWS client', {error: error});
+                }
+                this.client = client
+              });
```

Comment:
Your most recent commit added a function with code that looks like this but with repeats in `index.ts` Should these lines have been replaced with a call to `setClient()`?